### ISSUE TITLE
Health check

### ISSUE THREAD

**Issue by blackandred** [riotkit-org/docker-taiga#6]
I think a more precised health checking should be created that would check the backend and frontend setup, the connectivity with SMTP and possibly with LDAP.

**Comment by zicklag** [comment]
That would be a good feature request for Taiga actually, to provide a `/status` page. I've seen that in other software before such as JIRA. Then healthcheck scripts only need to do a `curl` to `/status` to ensure the health of the application.

Not that we couldn't implement it ourselves because I have no idea how long it would be before they worked something like that in.

**Comment by blackandred** [comment]
I agree, each application should implement that. If the application does
not implement an endpoint, then we can provide some basic checking.

**Comment by Leopere** [comment]
Even curl'ing an asset from the Taiga-Front like a favicon would be indicative of uptime.  I don't know what you'd query for Taiga-Back but I feel a curl to the API would be where to begin there.  Simply curling isn't perfect but it would at least show some degree of container health so if you want self-healing or dependencies you can have them rely on the run status of the two services.

**Comment by blackandred** [comment]
That's a good idea Leopere :)

**Comment by h44z** [comment]
in terms of container health checks i would recommend using https://github.com/aelsabbahy/goss.
this nice utility can check processes, api's and so on =)

then simply include a script:
```bash
#!/bin/bash

set -e

goss -g /app/goss.yaml validate --format json_oneline

exit 0
```

in the dockerfile:

```
HEALTHCHECK --interval=1m --timeout=10s \
    CMD /app/healthcheck.sh
```
